Index: webkit/glue/webthemeengine_impl_win.cc
===================================================================
--- glue/webthemeengine_impl_win.cc	(revision 29571)
+++ glue/webthemeengine_impl_win.cc	(working copy)
@@ -100,6 +100,17 @@
       draw_edges);
 
   canvas->endPlatformPaint();
+
+  // Add 255 to the region's alpha channel to compensate for how GDI sets it to 0
+  SkRect fillRect;
+  SkPaint fillPaint;
+  fillPaint.setXfermodeMode(SkXfermode::kPlus_Mode);
+  fillPaint.setColor(SkColorSetARGB(255, 0, 0, 0));
+  fillRect.set(
+    SkIntToScalar(native_rect.left), SkIntToScalar(native_rect.top), 
+    SkIntToScalar(native_rect.right), SkIntToScalar(native_rect.bottom)
+  );
+  canvas->drawRect(fillRect, fillPaint);
 }
 
 void WebThemeEngineImpl::paintTrackbar(
Index: app/gfx/native_theme_win.cc
===================================================================
--- app/gfx/native_theme_win.cc	(revision 29571)
+++ app/gfx/native_theme_win.cc	(working copy)
@@ -511,11 +511,14 @@
     }
 
     // TODO(maruel): Need to be fixed if get_theme_content_rect_ is NULL.
-    if (fill_content_area && get_theme_content_rect_) {
+    if (get_theme_content_rect_) {
       RECT content_rect;
       hr = get_theme_content_rect_(handle, hdc, part_id, state_id, rect,
                                    &content_rect);
-      FillRect(hdc, &content_rect, bg_brush);
+      if (fill_content_area)
+        FillRect(hdc, &content_rect, bg_brush);
+      else
+        *rect = content_rect;
     }
   } else {
     // Draw it manually.