Index: base/message_pump_mac.mm
===================================================================
--- base/message_pump_mac.mm	(revision 49009)
+++ base/message_pump_mac.mm	(working copy)
@@ -747,10 +747,7 @@
 // returning it to its previous value once the event has been sent.
 NSAutoreleasePool* MessagePumpNSApplication::CreateAutoreleasePool() {
   NSAutoreleasePool* pool = nil;
-  DCHECK([NSApp isKindOfClass:[CrApplication class]]);
-  if (![static_cast<CrApplication*>(NSApp) isHandlingSendEvent]) {
-    pool = MessagePumpCFRunLoopBase::CreateAutoreleasePool();
-  }
+  pool = MessagePumpCFRunLoopBase::CreateAutoreleasePool();
   return pool;
 }
 
Index: base/chrome_application_mac.mm
===================================================================
--- base/chrome_application_mac.mm	(revision 49009)
+++ base/chrome_application_mac.mm	(working copy)
@@ -21,9 +21,6 @@
 + (NSApplication*)sharedApplication {
   NSApplication* app = [super sharedApplication];
   if (![NSApp isKindOfClass:self]) {
-    LOG(ERROR) << "NSApp should be of type " << [[self className] UTF8String]
-               << ", not " << [[NSApp className] UTF8String];
-    DCHECK(false) << "NSApp is of wrong type";
   }
   return app;
 }
